curriculumBuilder/local_settings.py.example

```diff
@@ -17,3 +17,10 @@ CACHES = {
         'LOCATION': 'unique-snowflake',
     }
 }
+
+DATABASES = {
+    "default": {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+	'NAME': 'curriculumbuilder',
```

Comment:
some weird spacing here, but otherwise LGTM!

Reply:
thank you for catching! it was a tab 😱